packages/ember-simple-auth/addon/initializers/setup-session-restoration.js

```diff
@@ -1,27 +1,35 @@
 import { getOwner } from '@ember/application';
 
 export default function setupSessionRestoration(registry) {
-  const ApplicationRoute = registry.resolveRegistration
-    ? registry.resolveRegistration('route:application')
-    : registry.resolve('route:application');
+  const Router = registry.resolveRegistration ? registry.resolveRegistration('router:main') : registry.resolve('router:main');
```

Comment:
I'm somewhat sure that this is private API, and I think using `reopen()` is generally deprecated 🤔

Reply:
I Wasn't really thinking about doing any refactoring here, but yeah, EmberObject apis in general are deprecated.
I'm not sure what we can do about it right now, IMO the way forward would be to remove the initializer, and if we have any crucial setup then users should probably call some method in their application routes.